src/main/java/org/websoso/WSSServer/domain/UserNovel.java

```diff
@@ -54,6 +60,12 @@ public class UserNovel extends BaseEntity {
     @JoinColumn(name = "novel_id", nullable = false)
     private Novel novel;
 
+    @OneToMany(mappedBy = "userNovel", cascade = ALL, fetch = FetchType.LAZY)
```

Comment:
p2;
부모 엔티티로 자식 엔티티 생명주기를 관리하려고 한다면 orphanRemoval도 걸어주면 어떨까욥?
```suggestion
    @OneToMany(mappedBy = "userNovel", cascade = ALL, fetch = FetchType.LAZY, orphanRemoval = true)
```